Cargo.toml

```diff
@@ -42,16 +42,16 @@ wasmcloud-host = { path = "crates/wasmcloud-host" }
 actix-rt = "1.1.1"
 nats = "0.8.6"
 structopt = "0.3.21"
-nkeys = "0.0.11"
+nkeys = "0.1.0"
 
 [dev-dependencies]
-rmp-serde = "0.14.4"
+rmp-serde = "0.15.4"
 serde = {version = "1.0.117", features = ["derive"] }
 serde_bytes = "0.11.5"
 serde_json = "1.0.59"
 provider-archive = "0.4.0"
-redis = "0.17.0"
-reqwest = { version = "0.10"}
+redis = "0.19.0"
+reqwest = { version = "0.11"}
```

Comment:
This `reqwest` update is causing the tests to fail, potentially due to a mismatch in tokio runtime versions? If you revert to `0.10` the tests pass locally on my machine, so I'd bet that will help it here.